AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -2983,7 +2983,7 @@ moneylo.fr###gatsby-focus-wrapper > div[style*="position: fixed;"][style*="z-ind
 navigium.de#$?##placeholder > div.modal:has(div.modal-header > h5:contains(Cookies)) { display: none!important; }
 navigium.de#$#.modal-backdrop { display: none!important; }
 navigium.de#$#body { overflow: visible!important; }
-0calc.es,pietsmiet.de,olevod.com,einthusan.tv,tomsguide.com,techradar.com,cinemaxxl.de,raider.io,xboxdynasty.de,imgur.com,joe.ie,joe.co.uk##.qc-cmp2-container
+fly4free.pl,0calc.es,pietsmiet.de,olevod.com,einthusan.tv,tomsguide.com,techradar.com,cinemaxxl.de,raider.io,xboxdynasty.de,imgur.com,joe.ie,joe.co.uk##.qc-cmp2-container
```

Comment:
As I recall, hiding cookie notice causes that some elements are not loaded - https://github.com/AdguardTeam/AdguardFilters/issues/90747

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/143850000-ec6ed83a-b44e-4843-b37c-7982ec9864e9.png)

</details> 

So, I think that we could modify this rule:
https://github.com/AdguardTeam/AdguardFilters/blob/5e24d75d2c6f74bce0ebdec108ca58146d1f079f/AnnoyancesFilter/sections/cookies_specific.txt#L7515
to for example, something like this:
```
fly4free.pl#%#AG_onLoad(function(){if(-1==document.cookie.indexOf("euconsent-v2")){var a=new MutationObserver(function(){var b=document.querySelector('.qc-cmp2-container .qc-cmp2-summary-buttons button[mode="primary"]');b&&(a.disconnect(),b.click())});a.observe(document,{childList:!0,subtree:!0});setTimeout(function(){a.disconnect()},1E4)}});
```

Reply:
@jellizaveta please make changes as suggested by @AdamWr .